sql/executor_test.go

```diff
@@ -80,6 +81,32 @@ func TestExecutorTrainAndPredictDNNLocalFS(t *testing.T) {
 	})
 }
 
+func TestExecutorTrainAndPredictionDNNClassifierDENSE(t *testing.T) {
+	if getEnv("SQLFLOW_TEST_DB", "mysql") == "hive" {
+		t.Skip(fmt.Sprintf("%s: skip Hive test", getEnv("SQLFLOW_TEST_DB", "mysql")))
+	}
+	a := assert.New(t)
+	a.NotPanics(func() {
+		stream := Run(`SELECT * FROM iris.train_dense
+TRAIN DNNClassifier
+WITH
+n_classes = 3,
+hidden_units = [10, 20],
+EPOCHS = 200,
+BATCHSIZE = 10
+COLUMN NUMERIC(dense, 4)
```

Comment:
maybe use data `text_cn.train_processed` to test dense column with delimiter?

Reply:
I think one advantage of using `iris.train_dense` is we are able to check the training accuracy.

Reply:
I mean, both tests are needed to cover dense column with a delimiter and without should be helpful. Thanks!!